test/RPM/testdata/simple-test.spec

```diff
@@ -0,0 +1,30 @@
+# A haiku showing how permissive our current parser is.
+To prove our parser
+This is not in a comment
+No hashtag needed
+
+# Basic parsing
+BuildRequires: boost
+
+# Simple version text
+BuildRequires: cmake >= 2.8
+BuildRequires: jsoncpp <= 0.10.5
+BuildRequires: gtest > 6.4
+BuildRequires: libevent < 9.8.1
+BuildRequires: libvirt == 3.9.0
+BuildRequires: openssl = 1.23.45
+
+# Bad operator, will ignore version
+BuildRequires: xz >> 1.2.3
+
+# Lots of spaces, should be fine
+BuildRequires:   libxml2     >=   5.4.98
+
+# Complex version text (shouldn't change behavior)
+BuildRequires:  gmock = 1.6.0-1.aapps.el7
+
+# Ignored due to macro use
+BuildRequires:  cloudistics-core%{?_isa} = %{version}-%{release}
```

Comment:
Real world is slightly more complex, but even so, it's not really applicable here.  A common pattern is to have something like `package` with ISA-agnostic code, and `package-isa` with ISA-specific code.  There's a distinct possibly of giving out incorrect licenses since much of the ISA-specific code can be maintained and released by third-parties.  I don't know how common this is, but even with my limited experience with these packages, I've seen that pattern a few times.  Makes me think it's common enough.